### DISCUSSION TITLE
Plot file size increase during harvester?

### DISCUSSION THREAD

**Discussion by matteoaugelli** [Chia-Network/chia-blockchain#6773] [Q&A]
I have noticed that plots generated with _Chia_ all have a dimension of the **101.4 GiB**, but it can change a bit.

### Example:
```
$ ls -laS /media/chia-plots/*.plot | awk '{print $5 " - " $9}'
108910468955 - /media/chia-plots/plot-k32-2021-05-26-14-14-*********.plot
108909655961 - /media/chia-plots/plot-k32-2021-05-31-12-17-*********.plot
108905968689 - /media/chia-plots/plot-k32-2021-05-22-15-22-*********.plot
108896204476 - /media/chia-plots/plot-k32-2021-06-02-20-01-*********.plot
108896063477 - /media/chia-plots/plot-k32-2021-06-06-09-31-*********.plot
108878790034 - /media/chia-plots/plot-k32-2021-05-25-14-19-*********.plot
108873981592 - /media/chia-plots/plot-k32-2021-06-03-23-47-*********.plot
108865715455 - /media/chia-plots/plot-k32-2021-06-05-15-57-*********.plot
108860766838 - /media/chia-plots/plot-k32-2021-06-03-18-09-*********.plot
108860628229 - /media/chia-plots/plot-k32-2021-06-01-10-25-*********.plot
108860144993 - /media/chia-plots/plot-k32-2021-05-27-14-28-*********.plot
108859654929 - /media/chia-plots/plot-k32-2021-05-24-02-52-*********.plot
.....
108770418701 - /media/chia-plots/plot-k32-2021-05-24-14-45-*********.plot
108756092734 - /media/chia-plots/plot-k32-2021-06-06-09-32-*********.plot
108742752039 - /media/chia-plots/plot-k32-2021-05-21-02-37-*********.plot
```

This are the data obtained by querying the HDD, its 91 plot size varies from **108910468955** to **108742752039** bytes;  with a difference  of **167716916 byte** (159.9 MiB) between the plots.

To understand which is the best TiB to choose, setting the value of the magnitude of a plot to **0.0996094 TiB** I generated this table:

![unnamed](https://user-images.githubusercontent.com/18537052/122046502-48efac80-cddf-11eb-91c0-48886c915e18.png)
[click to see the bigger picture](https://user-images.githubusercontent.com/18537052/122044676-0f1da680-cddd-11eb-8a2f-f69bd638ffad.png)

Therefore it seems that the 8TB  is the best choice considering that the space wasted is around 50MB

It happened to me that on 2 HDDs of 8TB each I had 2 corrupted plots and when I checked their size was less than the nominal value of a plot.
In these two HDD where i had this issue now I have **72** plots, but there is no more space to store another one.

* When the _harvester_ is running is possible to change the size of the plots participating in the lotteries? 
* Does it need a reserved space to avoid the corruption of the plots?

Thanks in advance

**Comment by loppefaaret** [comment]
i suspect those 2 plots were corrupted on creation - or when moving them into place
as for optimal space usage, i myself is combining a mix a of k32 and k33, and plan out a drive layout with this tool:
https://plot-plan.chia.foxypool.io/
it is not possible to change the size of the .plot files, it is solely depending on the generation process as to what exact size they come out
